File: 1-12Quiz/cpp/1/main.cpp
#include <iostream>
using namespace std;

int readNumber(){
    int i;
    cout<<"Enter an integer "<<endl;
    cin>>i;
   
    return i;
}

void writeAnswer(int a, int b){
    cout<<"The answer is "<<a+b<<endl;
}

int main(){
    int a = readNumber();
    int b = readNumber();
    writeAnswer(a, b);

    return 0;
}

File: 1-12Quiz/ts/1/main.js
function readNumber() {
    return prompt("Enter a number ");
}
function writeAnswer(a, b) {
    console.log("The answer is " + (a + b));
}
(function main() {
    var a = parseInt(readNumber());
    var b = parseInt(readNumber());
    writeAnswer(a, b);
})();
